### PR TITLE
Webhooks and endpoint changes

### PR DESCRIPTION
**Pull request by MG4CE**

### CODE REVIEW

File: microservices/projects/handlers/tasks_constants.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package handlers
+
```
Comment (robotevan) [inline microservices/projects/handlers/tasks_constants.go:2]: const?

File: microservices/projects/handlers/tasks_handlers.go [hunk 2]
```diff
@@ -213,7 +214,60 @@ func (p *Projects) DeleteTask(w http.ResponseWriter, r *http.Request) {
 
 // EditTask edit the details of select task.
 func (p *Projects) EditTask(w http.ResponseWriter, r *http.Request) {
-	http.Error(w, "Not Implemented", http.StatusNotImplemented)
+	projectName := chi.URLParam(r, "project-name")
+	taskIDStr := chi.URLParam(r, "task-id")
```
Comment (robotevan) [inline microservices/projects/handlers/tasks_handlers.go:218]: url param or put in payload? Up to you

File: scripts/gitcollab.sh [hunk 3]
```diff
@@ -56,18 +56,20 @@ function test-integration() {
     source gitcollab_pyenv/bin/activate && pytest ./integration_tests/
 }
 
-function fetch_valid_containers() {
+function fetch_valid_containers() {\
```
Comment (robotevan) [inline scripts/gitcollab.sh:59]: should slash be here?

**Review by robotevan — CHANGES_REQUESTED** [review]

File: microservices/projects/handlers/tasks_handlers.go (hunk 2 above)
Comment (MG4CE) [reply microservices/projects/handlers/tasks_handlers.go:218]: Does not really matter can be easily changed if we want later on.

File: scripts/gitcollab.sh (hunk 3 above)
Comment (MG4CE) [reply scripts/gitcollab.sh:59]: prob not

File: microservices/projects/handlers/tasks_constants.go (hunk 1 above)
Comment (MG4CE) [reply microservices/projects/handlers/tasks_constants.go:2]: no

Comment (robotevan) [reply microservices/projects/handlers/tasks_constants.go:2]: image

File: microservices/projects/handlers/tasks_handlers.go (hunk 2 above)
Comment (robotevan) [reply microservices/projects/handlers/tasks_handlers.go:218]: image

File: scripts/gitcollab.sh (hunk 3 above)
Comment (robotevan) [reply scripts/gitcollab.sh:59]: remove 
image

**Review by robotevan — APPROVED** [review]

**Pull request merged**